$NetBSD: patch-aa,v 1.11 2013/10/20 18:00:57 joerg Exp $

* pthread flags portabilities
* avoid check for gettext>=0.11.5
* fixes to install all supported L10N man pages

--- configure.orig	2011-11-11 20:59:26.000000000 +0000
+++ configure
@@ -3345,8 +3345,6 @@ case $host_os in *\ *) host_os=`echo "$h
 		# Now this is against autoconf recommendation that configure should not modify CPPFLAGS and LDFLAGS
 		# However, these values in NetBSD are required even to run the tests, and this is the easiest way to do it.
 		# Still, we prepend them, instead of adding, so the user may override them.
-		CPPFLAGS="-I/usr/pkg/include $CPPFLAGS"
-		LDFLAGS="-R/usr/pkg/lib -L/usr/pkg/lib $LDFLAGS"
 		;;
 	*irix*)
 		SYS=irix
@@ -3364,7 +3362,8 @@ case $host_os in *\ *) host_os=`echo "$h
 $as_echo_n "checking if this is a FreeBSD 4 or earlier system... " >&6; }
 		if test x"`uname -s`" = xFreeBSD && test 0`uname -r | cut -c 1` -lt 5; then :
 
-			MULELDFLAGS="$MULELDFLAGS -pthread"
+			MULELDFLAGS="$MULELDFLAGS ${PTHREAD_LDFLAGS} ${PTHREAD_LIBS}"
+			MULECFLAGS="$MULECFLAGS ${PTHREAD_CFLAGS}"
 			{ $as_echo "$as_me:${as_lineno-$LINENO}: result: yes" >&5
 $as_echo "yes" >&6; }
 
@@ -13760,7 +13759,7 @@ $as_echo "#define HAVE_DCGETTEXT 1" >>co
   INCINTL=-I\${top_builddir}/intl
 fi
 
-	if test x$USE_NLS = xyes; then :
+	if test xno = xyes; then :
 
 	{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for autopoint" >&5
 $as_echo_n "checking for autopoint... " >&6; }
@@ -13800,7 +13799,7 @@ $as_echo_n "checking for requested langu
 
 	if test ${enable_amule_daemon:-no} = yes; then :
 
-		if test -z "$LINGUAS"; then :
+		if test -z "$LINGUAS" -o "$LINGUAS" = "%UNSET%"; then
   AMULE_DAEMON_MANPAGES=`ls -1 ${srcdir}/docs/man/amuled.* | sed -e 's:.*/::g'`
 else
 
@@ -13818,7 +13817,7 @@ fi
 
 	if test ${enable_amulecmd:-no} = yes; then :
 
-		if test -z "$LINGUAS"; then :
+		if test -z "$LINGUAS" -o "$LINGUAS" = "%UNSET%"; then
   AMULECMD_MANPAGES=`ls -1 ${srcdir}/docs/man/amulecmd.* | sed -e 's:.*/::g'`
 else
 
@@ -13836,7 +13835,7 @@ fi
 
 	if test ${enable_webserver:-no} = yes; then :
 
-		if test -z "$LINGUAS"; then :
+		if test -z "$LINGUAS" -o "$LINGUAS" = "%UNSET%"; then
   WEBSERVER_MANPAGES=`ls -1 ${srcdir}/docs/man/amuleweb.* | sed -e 's:.*/::g'`
 else
 
@@ -13854,7 +13853,7 @@ fi
 
 	if test ${enable_amule_gui:-no} = yes; then :
 
-		if test -z "$LINGUAS"; then :
+		if test -z "$LINGUAS" -o "$LINGUAS" = "%UNSET%"; then
   AMULE_GUI_MANPAGES=`ls -1 ${srcdir}/docs/man/amulegui.* | sed -e 's:.*/::g'`
 else
 
@@ -13872,7 +13871,7 @@ fi
 
 	if test ${enable_cas:-no} = yes; then :
 
-		if test -z "$LINGUAS"; then :
+		if test -z "$LINGUAS" -o "$LINGUAS" = "%UNSET%"; then
   CAS_MANPAGES=`ls -1 ${srcdir}/src/utils/cas/docs/cas.* | sed -e 's:.*/::g'`
 else
 
@@ -13890,7 +13889,7 @@ fi
 
 	if test ${enable_wxcas:-no} = yes; then :
 
-		if test -z "$LINGUAS"; then :
+		if test -z "$LINGUAS" -o "$LINGUAS" = "%UNSET%"; then
   WXCAS_MANPAGES=`ls -1 ${srcdir}/src/utils/wxCas/docs/wxcas.* | sed -e 's:.*/::g'`
 else
 
@@ -13908,7 +13907,7 @@ fi
 
 	if test ${enable_ed2k:-yes} = yes; then :
 
-		if test -z "$LINGUAS"; then :
+		if test -z "$LINGUAS" -o "$LINGUAS" = "%UNSET%"; then
   ED2K_MANPAGES=`ls -1 ${srcdir}/docs/man/ed2k.* | sed -e 's:.*/::g'`
 else
 
@@ -13926,7 +13925,7 @@ fi
 
 	if test ${enable_alc:-no} = yes; then :
 
-		if test -z "$LINGUAS"; then :
+		if test -z "$LINGUAS" -o "$LINGUAS" = "%UNSET%"; then
   ALC_MANPAGES=`ls -1 ${srcdir}/src/utils/aLinkCreator/docs/alc.* | sed -e 's:.*/::g'`
 else
 
@@ -13944,7 +13943,7 @@ fi
 
 	if test ${enable_alcc:-no} = yes; then :
 
-		if test -z "$LINGUAS"; then :
+		if test -z "$LINGUAS" -o "$LINGUAS" = "%UNSET%"; then
   ALCC_MANPAGES=`ls -1 ${srcdir}/src/utils/aLinkCreator/docs/alcc.* | sed -e 's:.*/::g'`
 else
 
@@ -13962,7 +13961,7 @@ fi
 
 	if test ${enable_monolithic:-yes} = yes; then :
 
-		if test -z "$LINGUAS"; then :
+		if test -z "$LINGUAS" -o "$LINGUAS" = "%UNSET%"; then
   MONOLITHIC_MANPAGES=`ls -1 ${srcdir}/docs/man/amule.* | sed -e 's:.*/::g'`
 else
 
